### ISSUE TITLE
Problem with dimensions in SVG file

### ISSUE THREAD

**Issue by jkramarz** [meerk40t/meerk40t#94]
When importing [simple SVG](https://pastebin.com/ZVWG6Pck) file saved from Inkscape:
![rendered svg](https://i.imgur.com/Bs7DMH1.png)
I'm getting quite unexpected result:
![same file opened in meerk40t](https://i.imgur.com/geEDiaV.png)

**Comment by tatarize** [comment]
There were a number of small but pressing bugs in the svg reading. I've spent the last month and some change getting all of them fixed. There's a couple minor deficits most of this stuff should be done and work for basically all svg.elements now. See Pull Request #90 for the version working with the new and corrected code. Here's your result in the new code.

![meerk-new](https://user-images.githubusercontent.com/3302478/71377902-db144d00-257a-11ea-9b19-816cbcf04a1d.png)
![meerk-new2](https://user-images.githubusercontent.com/3302478/71377964-ff702980-257a-11ea-98d0-41bf1d63dcbd.png)


You'll see it hits correctly on length and height. And should be fine on the x and y too. There's a bunch of different ways svgs specify that stuff and I went through and all of them. To the point of passing most all the W3C_SVG_11_TestSuite rather nicely.

Basically there was a serious need to get this right so that I can copy whisperer's workflow where people will very often do all their alignment work for images in inkscape and then reload the svg in the laser cutting program so I needed all that alignment stuff to correctly work.
